### PR TITLE
Release: v2024.11.7

### PR DESCRIPTION
**Pull request by taylorjdawson**
## Did you add a changeset?

If updating one of our packages, you'll likely need to add a changeset to your PR. To do so, run [`pnpm changeset`](https://pnpm.io/using-changesets#adding-new-changesets). `pnpm changeset` will generate a file where you should write a human friendly message about the changes. Note how this ([example](https://github.com/tkhq/sdk/blob/b409cd06790f011bf939adcf0755499b8e7497ae/.changeset/extra-http-exports.md?plain=1#L1)) includes the package name (should be auto added by the command) along with the type of [semver change (major.minor.patch)](https://semver.org/) (which you should set).

These changes will be used at release time to determine what packages to publish and how to bump their version. For more context see [this comment](https://github.com/tkhq/sdk/pull/67#issuecomment-1568838440).

### CODE REVIEW

**Review by andrewkmin — APPROVED** [review]

File: packages/wallet-stamper/CHANGELOG.md [hunk 1]
```diff
@@ -1,29 +1,70 @@
...
 
 ## 0.0.2
 
 ### Patch Changes
 
 - 68a14dd: Initial release! 🎉
+
+```
+
+```
```
Comment (r-n-o) [inline packages/wallet-stamper/CHANGELOG.md:70]: That seems unintentional

File: examples/kitchen-sink/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# @turnkey/kitchen-sink
+
+## 0.1.2
+
+### Patch Changes
+
+- Updated dependencies [8bea78f]
+  - @turnkey/sdk-browser@1.10.0
+  - @turnkey/sdk-react@1.2.0
```
Comment (r-n-o) [inline examples/kitchen-sink/CHANGELOG.md:9]: do we need changelogs for all examples? I don't think this adds much but I'm okay either way

Comment (moe-dev) [reply examples/kitchen-sink/CHANGELOG.md:9]: I've removed the changelogs for examples in the last release

Comment (taylorjdawson) [reply examples/kitchen-sink/CHANGELOG.md]: Removed!

**Review by r-n-o — APPROVED** [review]

**Pull request merged**